Open Waters Swimming Conference Website

lmunozm1702 opened the pull request:
## Module 1 - Capstone project:

## I develop the website for the next Open Waters Swimming Conference.

The website has 2 pages, one Landing Page and an About Page.

Both pages are responsive for mobiles and desktop devices. For desktop, I set 2 breakpoints, the first between 678 and 1024 px and the second over the 1024 px width.

In the Landing Page, the main sections, for the Conference Program and the Speakers are developed using javascript and reading the data from javascript data files.

The styling was doit using CSS and Bootstrap.

### The videos presenting the functionality and the technical detail are:

https://www.loom.com/share/bfc6469d38ff4e9688891ea62e13ded8
https://www.loom.com/share/9adb526c207844fcad92e863283d5aaf


I need to make 2 videos because I can't show Visual Studio Code in the same video displaying the webpage on the chrome browser.

lmunozm1702 committed:
Add README.md file

lmunozm1702 committed:
Fix linter issues

Bondok6 commented on README.md:
```diff
@@ -1,2 +1,68 @@
+
+https://lmunozm1702.github.io/module1-capstone/
+
+## Getting Started
+
+### Prerequisites
+
+### Setup
+
+### Install
```
- [x] I think it would be better to add some instructions or steps on how to install and run your project in `Getting Started` section, such as `git clone <project url>`, `cd <project name>`, ... 💯 

- [x] Kindly, Remove the rest of empty sections in your `README` 👍

> We should always be sure our repositories are documented professionally, please fix broken links on the readme file.

Bondok6 commented on README.md:
```diff
@@ -1,2 +1,68 @@
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```
- [x] I think it would be better to remove this `_NOTE ... `  from your README file 💯

> We should always be sure our repositories are documented professionally, please fix broken links on the readme file.

Bondok6 reviewed (changes requested):
## Hi @lmunozm1702 🙋‍♂️, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<img src="https://media0.giphy.com/media/31Q36EHJ7C5ZoIMw79/giphy.gif"/>

## Highlights
- No Linter errors ✅
- Correct GitHub flow ✅
- Great job using semantic HTML5 tags ✅
- Deploy using gh-pages ✅
- Good video presentation ✅


## Required Changes ♻️
- [x] According to the requirements and the original design, You should follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including Layout: composition and space between elements., So kindly remove the left and right space on desktop version to look similar to the original design 👍💯

| Your Design  | Original Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/196855632-a1ce5f7b-efb0-4517-942a-d7a2a0868984.PNG" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/196855714-ea0301c6-db68-49d1-a270-0e98129ca62f.png" width="500" /> |


- [x] As I mentioned in the previous comment and according to the requirements and the original design, You should follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including Colors, Typography, and Layout. So kindly update the button/link in `main program` section should look like the original design on desktop version 👍

| Your Design  | Original Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/196855892-51ceaa26-8ac4-4cc2-8a97-cbbd1b606ab7.png" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/196855979-d5886cb3-e733-4bcc-a611-03f420ad61d1.PNG" width="500" /> |

- [x] Great job implementing the functionality of  `see more` button 💯 However,  The `see more` button should only appear on the mobile version, so please display all the speakers on the desktop version 👍

| Your Design  | Original Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/196856641-55793e31-a224-48d0-9fb4-ff35f67d619a.png" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/196856492-bc902aad-f12b-4c4a-864a-0c8893e52195.png" width="500" /> |


- [x] in About page, I think you forgot to add the middle section, it should contains a section header, a divider, a paragraph of text, and an image or large logo, so Please add it 💯

![4](https://user-images.githubusercontent.com/45231276/196857301-f5e35f60-a5c4-44cf-abd1-a5a88096f768.PNG)


- [x] _Check the comments under the review._ ⬇



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

lmunozm1702 committed:
Fix Code Review Issues

Update about.html, index.html, main.js, README.md and .gitignore to fix issues detected on the Code Review.

lmunozm1702 committed:
Fix code review issues

Remove bootstrap classes for video elements.

lmunozm1702 committed:
Fix linter errors

lmunozm1702 commented:
Hi @Bondok6 All issues are solved!

For the section missed on the About page, this event doesn't have any icon, like the section on the template, I replace them with a short explanation about open waters swimming and I add a video of that.

Bondok6 commented:
Hi @lmunozm1702 
For About page it's ok you can replace it with any information 🤝
Great job implementing the changes 🎉 You can re-submit, and another reviewer will review your project 👍

Kamba56 commented on about.html:
```diff
@@ -19,21 +19,23 @@
-                <li class="nav-item">|</li>
-                <li class="nav-item">Logout</li>
-            </ul>
+            <div class="top-nav-container">
+                <ul id="top-navbar-list" class="nav justify-content-end gap-2">
+                    <li class="nav-item"><i class="fa-brands fa-facebook"></i></li>
+                    <li class="nav-item">|</li>
+                    <li class="nav-item"><i class="fa-brands fa-twitter"></i></li>
+                    <li class="nav-item">|</li>
+                    <li class="nav-item">English</li>
+                    <li class="nav-item">|</li>
+                    <li class="nav-item">My Page</li>
+                    <li class="nav-item">|</li>
+                    <li class="nav-item">Logout</li>
+                </ul>
+            </div>
```
- [Optional] consider using an indentation of two spaces in all your documents for readability. This is a convention and also [best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

Kamba56 reviewed (approved):
Hi @lmunozm1702, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 
👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
